games/d.yaml

```
@@ -1046,6 +1046,26 @@
   - https://dreerally.com/wp-content/uploads/2016/12/Captura3-300x236.jpg
   - https://dreerally.com/wp-content/uploads/2016/12/Captura4-300x233.jpg
 
+- name: Dragon Alpha
+  originals:
+  - Dragon Alpha
+  type: official
+  repo: 'https://github.com/jaredplumb/Dragon-Alpha'
+  development: halted
+  status: playable
+  content: open
+  langs:
+  - C++
+  frameworks:
+  - DrawSprocket
```

```suggestion
  - Game Sprockets
```
Not familiar with this API but it sounds more general than DrawSprocket, similar to DirectX
Please also add this to dangerfile.js to silence the frameworks warning

---

Yeah, DrawSprocket is one component of Game Sprockets, which was an older Mac OS (between Classic and OSX) framework that meant to be an equivalent to DirectX. I wasn't sure if I should use "DrawSprocket" or  "Game Sprockets" because they only seemed to be using the DrawSprocket component, but I guess it would make sense in a similar situation to say "DirectX" instead of "DirectInput" or "DirectSound".

Also, should I add Game Sprockets to dangerfile.js as "Game Sprockets" or "GameSprockets"? The former seems to be used more by third parties (including Wikipedia), while the latter seems to be what Apple used themselves. Or am I able to add both versions?